### DISCUSSION TITLE
Is there a way to group menu items?

### DISCUSSION THREAD

**Discussion by silver8ack** [aminalaee/sqladmin#510] [General]
Flask-Admin allows a 'category' property when defining a view to allow grouping in the menu.  Does sqladmin have something similar?

**Comment by aminalaee** [comment]
No, not yet.
I will add it to my todo-list.

**Reply by wangxin688** [reply]
wonderful features. if many modelViews are added to menu. if PR is accepted, I could do some research on this feature

**Reply by aminalaee** [reply]
The category option was added recently. Do you mean to build on top of that?

**Reply by wangxin688** [reply]
sorry, after check the docs I find `category` fields has been added.

**Reply by wangxin688** [reply]
after checking source code of `CategaryMenu`, `icon` field is contained here, may I know why `category_icon` is missed in `ModelView`?

**Reply by aminalaee** [reply]
It might have been a mistake, feel free to double-check and add it
